recipes/nestml/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "nestml" %}
+{% set version = "7.0.2" %}
+{% set build = 0 %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nest/nestml/archive/v{{ version }}.tar.gz
+  sha256: d2cc7b119b2b2eda98ac1b148d4e947f77fef2fdce563ff829b7d7185ce81be3
+
+build:
+  number: {{ build }}
+  entry_points:
+    - nestml = pynestml.frontend.pynestml_frontend:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.8.2
+    - sympy
+    - scipy
+    - antlr4-python3-runtime ==4.13
+    - setuptools
+    - jinja2 >=2.10
+    - typing  # [py<35]
```

Comment:
Remove this dependency and add a matching python lower bound. Then you can make the whole package noarch.